source/developers/internationalization/backend_localization.markdown

```diff
@@ -11,9 +11,41 @@ ha_release: 0.64
 ---
 
 ## {% linkable_title Translation Strings %}
-Platform translation strings are stored as JSON in the [home-assistant](https://github.com/home-assistant/home-assistant) repository. These files must be located adjacent to the component/platform they belong to. They are named `strings.<component/platform name>.json`, unless the component being translated exists in its own directory, in which case the file is simply named `strings.json` in that directory. This file will contain the different strings that will be translatable. Currently only states are supported by the frontend.
+Platform translation strings are stored as JSON in the [home-assistant](https://github.com/home-assistant/home-assistant) repository. These files must be located adjacent to the component/platform they belong to. They are named `strings.<component/platform name>.json`, unless the component being translated exists in its own directory, in which case the file is simply named `strings.json` in that directory. This file will contain the different strings that will be translatable.
```

Comment:
Should we completely drop support for  `strings.<component/>.json` for components and only allow it in their own directories?

Reply:
:+1: I'm good with that.